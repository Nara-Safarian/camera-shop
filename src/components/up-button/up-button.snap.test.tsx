import { render } from '@testing-library/react';
import UpButton from './up-button';

test('UpButton snapshot should be rendered correctly', () => {
  const {container} = render(
    <UpButton />
  );
  expect(container).toMatchSnapshot();
});
