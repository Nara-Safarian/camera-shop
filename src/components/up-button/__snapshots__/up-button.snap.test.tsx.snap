// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UpButton snapshot should be rendered correctly 1`] = `
<div>
  <button
    class="up-btn"
  >
    <svg
      aria-hidden="true"
      height="18"
      width="12"
    >
      <use
        xlink:href="#icon-arrow2"
      />
    </svg>
  </button>
</div>
`;
